src/sdk/contracts/GovernanceStakingContract.ts

```diff
@@ -16,8 +16,11 @@ import { Ether } from '@uniswap/sdk-core'
  * @param {string?} address Deployed contract address in given chain ID.
  * @constructor
  */
-export function governanceStakingContract(web3: Web3, address?: string) {
-    address = address ?? G$ContractAddresses(SupportedChainId.FUSE, 'GovernanceStaking')
+export function governanceStakingContract(web3: Web3, address?: string, isv2?: boolean) {
```

Comment:
this specifically has the "address" as an input, there's no need for isV2, just supply the address directly from the calling method